bot/engine/src/main/kotlin/admin/dialog/DialogRating.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2017/2021 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.bot.admin.dialog
+
+/**
+ * A dialog is a conversation between users and bots.
```

Comment:
Nice to have some documentation there, but it explains what a Dialog is, not what a DialogRating is, does it?

Reply:
resolved